src/components/playground/CommentContent.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+import ReactEmoji from 'react-emoji';
+
+@connect(state => state.playground)
+@Radium
+class CommentContent extends React.Component {
+
+  filterContent(content) {
+    var tagSearchPattern = /{(.+)}(.*){\/(\1)}/;
```

Comment:
would you mind commenting this function up? Some poor soul is going to come in here and not know what this regex does like 2 years from now.